user_tools/docs/user-tools-databricks-aws.md

```diff
@@ -0,0 +1,156 @@
+
+## Prerequisites
+
+### 1.Databricks CLI
+- Install the Databricks CLI. Follow the instructions on [Databricks CLI setup & documentation](https://docs.databricks.com/dev-tools/cli/index.html).
+- Set the configuration settings and credentials of the Databricks CLI:
+  - Set up authentication by following these [instructions](https://docs.databricks.com/dev-tools/cli/index.html#set-up-authentication)
+  - Test the authentication setup by following these [instructions](https://docs.databricks.com/dev-tools/cli/index.html#test-your-authentication-setup)
+  - Verify that the access credentials are stored in the file `~/.databrickscfg` on Unix, Linux, or macOS, or `%USERPROFILE%\.databrickscfg` on Windows.
+
+### 2.AWS CLI
+
+- Install the AWS CLI version 2. Follow the instructions on [aws-cli-getting-started](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html).
+- Set the configuration settings and credentials of the AWS CLI by creating `credentials` and `config` files as described in [aws-cli-configure-files](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
+
+### 3.RAPIDS tools
+
+- Spark event logs:
+  - The RAPIDS tools can process Apache Spark CPU event logs from Spark 2.0 or higher (raw, .lz4, .lzf, .snappy, .zstd)
+  - For `qualification` commands, the event logs need to be archived to an accessible S3 folder.
+
+### 4.Install the package
+
+- Install `spark-rapids-user-tools` with python [3.8, 3.10] using:
+  - pip:  `pip install spark-rapids-user-tools`
+  - wheel-file: `pip install <wheel-file>`
+  - from source: `pip install -e .`
+- verify the command is installed correctly by running
+  ```bash
+    spark_rapids_user_tools databricks_aws --help
+  ```
+
+### 5.Environment variables
+
+Before running any command, you can set environment variables to specify configurations.
+- RAPIDS variables have a naming pattern `RAPIDS_USER_TOOLS_*`:
+  - `RAPIDS_USER_TOOLS_CACHE_FOLDER`: specifies the location of a local directory that the RAPIDS-cli uses to store and cache the downloaded resources. The default is `/tmp/rapids_user_tools_cache`.  Note that caching the resources locally has an impact on the total execution time of the command.
+  - `RAPIDS_USER_TOOLS_OUTPUT_DIRECTORY`: specifies the location of a local directory that the RAPIDS-cli uses to generate the output. The wrapper CLI arguments override that environment variable (`--local_folder` for Qualification).
+- For AWS CLI, some environment variables can be set and picked by the RAPIDS-user tools such as: `AWS_PROFILE`, `AWS_DEFAULT_REGION` and `AWS_CONFIG_FILE`. See the full list of variables in [aws-cli-configure-envvars](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).
+
```

Comment:
This list is missing the DB defined variables. We should extend this list to include any env-variable used in `databricks_aws-configs.json` such as: `DATABRICKS_CONFIG_FILE`